server/src/config.rs

```diff
@@ -46,4 +47,25 @@ pub struct ServerConfig {
     /// The p2p keypair file. A random one will be generated if not provided.
     #[clap(long = "p2p-keypair", env = "MC_P2P_KEYPAIR")]
     pub p2p_keypair_path: Option<PathBuf>,
+    /// This is a vector corresponding to a map from token ids to the minimum
```

Comment:
Nit: Every other field in this struct has a newline between it and the previous field.

```suggestion

        /// This is a vector corresponding to a map from token ids to the minimum
```